docs/basics/userguide/save_load.md

```diff
@@ -149,9 +149,130 @@ Success!
 
 As you can see, we haven't re-fit the experiment, and the q-table is the same as the one previously saved (and the Agent can finish the environment).
 
-## Other informations
+## Other information
 
 The `save` and `load` can be useful for :
 - you want to train your agent on a computer, and test/use it on others.
 - you have a long training, and you want to do some 'checkpoints'.
 - you want to do the training in more than once (only if your agent has "fit(x) then fit(y), is the same as fit(x+y)")
+
+
+## How to save/load an agent only?
+
+<span>&#9888;</span> We highly recommend to use the save/load with the `ExperimentManager` to have all the information. But if you need to save the agent for a specific case, you can do so : <span>&#9888;</span>
```

Comment:
For people not used to the difference between ExperimentManager and Agent, this sentence may be hard to understand.

Reply:
For the people not used to the difference between ExperimentManager and Agent, maybe they should go with the ExperimentManager only?